### PR TITLE
SEO-171351 ASP.NET Core Bing reports

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan ,

Have changed description, title, h1, redirect links and image alt.

Regards,
Yvone.

### CODE REVIEW

File: aspnet-core/TimePicker/Overview.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-description: overview
+title: Overview of ASP.NET Core TimePicker Control | Syncfusion
+description: Learn here more about Overview of Syncfusion Essential ASP.NET Core TimePicker Control, its elements, and more.
 platform: aspnet-core
 control: TimePicker
 documentation: ug
 ---
 
-# TimePicker
+# Overview of ASP.NET Core TimePicker Control
```
Comment (gowthamanthavasiyappan) [inline aspnet-core/TimePicker/Overview.md:10]: Remove control word

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: aspnet-core/TimePicker/Overview.md (hunk 1 above)
Comment (Yvone-Atieno) [reply aspnet-core/TimePicker/Overview.md:10]: removed

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**